microsoft-edge/devtools-guide-chromium/performance/reference.md

```diff
@@ -0,0 +1,1238 @@
+
+* Idle frame (white). No changes.
+
+* Frame (green). Rendered as expected and in time.
+
+* Partially presented frame (yellow with a sparse wide dash-line pattern).  Microsoft Edge did its best to render at least some visual updates in time.  For example, in case the work of the main thread of the renderer process (canvas animation) is late but the compositor thread (scrolling) is in time.
+
+* Dropped frame (red with a dense solid-line pattern).  Microsoft Edge can't render the frame in reasonable time.
+
+<!-- ![Hovering over a partially presented frame](todo: hovering-a-partially-pre-123.png) -->
+
+<!-- The above example shows a tooltip when you hover over a partially presented frame. -->
+
+Click a frame to view more information about the frame in the **Summary** tab at the bottom of the **Performance** tool.  DevTools outlines the selected frame in blue:
+
+![Viewing a frame in the Summary tab](./reference-images/frames-summary.png)
+
+
+<!-- ------------------------------ -->
+#### View network requests
+<!-- https://developer.chrome.com/docs/devtools/performance/reference#network -->
+<!-- upstream walks through a different demo page, so section content differs -->
+
+Expand the **Network** section to view a waterfall of network requests that occurred during the recording:
+
+![The Network section](./reference-images/network.png)
+
+Click on a request to view more information about it in the **Summary** tab:
+
+![More information about the photo-gallery request in the Summary tab](./reference-images/summary-network.png)
+
+Requests in the **Network** section are color-coded as follows:
+
+* Blue background: HTML request.
+* Purple background: CSS request.
+* Dark yellow background: JS request.
+* Green background: image request.
+
+Requests have squares in the top-left corner:
+* A darker-blue square in the top-left of a request means it's a higher-priority request.
+* A lighter-blue square means lower-priority.
+
+For example, in the previous figure, the **photo-gallery** request, in the top left corner of the **Network** section is higher-priority.
+
+Requests might have lines on the left and right sides, and their bars might be split into two colors.  Here's what these lines and colors represent:
+
+* The left line is everything up to the `Connection Start` group of events, inclusive.  In other words, it's everything before `Request Sent`, exclusive.
+
+* The light portion of the bar is `Request Sent` and `Waiting (TTFB)` for server response.
+
+* The dark portion of the bar is `Content Download`.
+
+* The right line is essentially time spent waiting for the main thread.  This isn't represented in the **Timing** tab.
+
+
+<!-- ------------------------------ -->
+#### View memory metrics
+<!-- https://developer.chrome.com/docs/devtools/performance/reference#memory -->
+
+Select the **Memory** checkbox to view memory metrics in a performance recording:
+
+![The Memory checkbox](./reference-images/memory-highlight.png)
```

Comment:
This image, and the next one, are broken in the internal preview.

Reply:
@captainbrosset  pls try:

1. go to internal preview anchor:
https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/performance/reference?branch=pr-en-us-3378#view-memory-metrics
2. In Edge, **F12** (DevTools) > Right-click **Refresh** > **Empty cache and hard refresh** > **F12**.

external preview anchor:
https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/perf-sync/microsoft-edge/devtools-guide-chromium/performance/reference.md#view-memory-metrics